### ISSUE TITLE
Update schema bug

### ISSUE THREAD

**Issue by den1k** [juji-io/datalevin#96]
When, after updating the schema using `d/update-schema`, an entity with an a new attribute is transacted both attributes end up having the same `:db/aid` which causes overwrites in the indexes.

```clojure
(defonce schema-bug-conn
  (d/create-conn
    "data/schema-bug-conn"
    {}))

(d/update-schema schema-bug-conn {:user/id {:db/unique    :db.unique/identity
                                            :db/valueType :db.type/string}})

(d/transact! schema-bug-conn
             [{:user/id       "foo"
               :user/some-edn {:foo :bar}}])

(d/schema schema-bug-conn)

{:db/ident #:db{:unique :db.unique/identity, :valueType :db.type/keyword, :aid 0},
 :user/id #:db{:unique :db.unique/identity, :valueType :db.type/string, :aid 1}, ;; <- :aid 1
 :user/some-edn #:db{:aid 1}} ;; <- :aid also 1

(d/datoms @schema-bug-conn :ave)
=> [#datalevin/Datom[1 :user/some-edn {:foo :bar}]
    #datalevin/Datom[1 :user/some-edn "foo"]]               ;; <- wrong value
```

**Comment by den1k** [comment]
It looks like the issue is right here: the aid is only incremented after being summed with max-aid. Starting it `i` at `1` should solve this.

https://github.com/juji-io/datalevin/blob/b3468e7271426be1c37ecff3f5c90431086f1954/src/datalevin/storage.cljc#L56-L67

**Comment by den1k** [comment]
actually, I just tried updating this. `update-schema` is not the problem. It's something the combination of using `update-schema` and then transactions with implicit new attrs.

**Comment by den1k** [comment]
Fixed in #96. @huahaiy if this is GTM could you cut a new release?
